src/stylesheets/components/Search/Search.sass

```
@@ -1,47 +1,47 @@
 .search
-  @include flex($direction: column)
-  animation: showSearch 0.3s linear forwards
-  background-color: $color-white
-  left: 0
-  position: absolute
-  top: -160px
-  width: 100%
-  z-index: $z-base
+    @include flex($direction: column)
+    animation: showSearch 0.3s linear forwards
+    background-color: $color-white
+    left: 0
+    position: absolute
+    top: -160px
```

```suggestion
    top: 0
```

Inside `showSearch` animation you should animate from `translateY(-100%)` to `translateY(0)`. This way you avoid magic numbers, and your animation would work regardless of the search height